README.md

```diff
@@ -22,3 +22,48 @@ ChRIS is comprised of several components and services, and it is designed to hav
   * [pfioh](https://github.com/FNNDSC/pfioh) - Handles input and output of image data into a data center
   * [pman](https://github.com/FNNDSC/pman) - Handles instantiating and checking the status of image processing jobs
   * Image Plugin(s) - An image processor. Turns raw image data into something meaningful for the radiologists/doctors. Ex: [FreeSurfer](https://surfer.nmr.mgh.harvard.edu/)
+
+## Usecase
+### Multi-Party Computation
+
+As we know that ChRIS (Children’s Research Integration System) is a web-based medical image platform that allows for various forms of medical image processing (for example, MRIs). To run medical image processing, medical researcher/analyst programs the alogorithm that needs to be executed in form of "plugin". The researcher/analyst submits the plugin along with the data on which the alogorithm is applied via ChRIS store. pfcon, running at BCH datacenter, sends the data via pfioh and the alorithm to be applied on the data to pman. pman and pfioh run in a different datacenter managed by OpenShift on top of OpenStack. pman translates the algorithm programmed in the plugin as Kubernetes job. To add scale to medical image processing, pman divides the job amongst several Kubernetes job pods that run in parallely. OpenShift provides container platform for seamless orchestration and resource management on the cluster.
```

Comment:
ChRIS Research Integration System (also change the other references in this file)